<?xml version="1.0" ?>
<launch>
<!--把 odom坐标系和map坐标系当成世界坐标系 -->
  <!-- make map and odom coincident for this demo -->
  <!-- <node name="odom_to_map"  pkg="tf" type="static_transform_publisher" output="screen"  args="0 0 0 0 0 0  /odom /map 200" /> -->

  <!-- <param name="use_sim_time" value="true" /> -->
  <!-- load Pronto config settings -->
 <!-- <arg name="fovis" default="true" /> -->
  <!-- launch the FOVIS node -->
 <!-- <include if="$(arg fovis)" file="$(find fovis_ros)/launch/realsense_simpledog.launch" >
    <arg name="output" value="screen"/>
  </include> -->

  <!--<node name="pose_means" pkg="pronto_laikago" type="pose_means" output="screen" /> -->
  <!-- launch the Pronto laikago node -->
  <node name="laikago_state" pkg="pronto_laikago" type="laikago_test" output="screen" />


  <!-- tf_publisher -->
  <include file="$(find pronto_laikago)/launch/pose_to_tf.launch"> 
  </include> 
</launch>
